### PR TITLE
3.0: Store LastUpdatedTime of compute fleet status in dynamodb table

### PR DESCRIPTION
**Pull request by hanwen-pcluste**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/slurm_plugin/clustermgtd.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ def is_stop_in_progress(status):
 class ComputeFleetStatusManager:
     COMPUTE_FLEET_STATUS_KEY = "COMPUTE_FLEET"
     COMPUTE_FLEET_STATUS_ATTRIBUTE = "Status"
+    LAST_UPDATED_TIME_ATTRIBUTE = "Last_updated_time"
```
Comment (demartinofra) [inline src/slurm_plugin/clustermgtd.py:82]: let's follow CamelCase convention here: `LastUpdatedTime`

File: src/slurm_plugin/clustermgtd.py [hunk 2]
```diff
@@ -107,7 +108,11 @@ def get_status(self, fallback=None):
     def update_status(self, current_status, next_status):
         try:
             self._table.put_item(
-                Item={"Id": self.COMPUTE_FLEET_STATUS_KEY, self.COMPUTE_FLEET_STATUS_ATTRIBUTE: str(next_status)},
+                Item={
+                    "Id": self.COMPUTE_FLEET_STATUS_KEY,
+                    self.COMPUTE_FLEET_STATUS_ATTRIBUTE: str(next_status),
+                    self.LAST_UPDATED_TIME_ATTRIBUTE: str(datetime.now()),
```
Comment (demartinofra) [inline src/slurm_plugin/clustermgtd.py:114]: let's always compute the datetime in UTC: `datetime.now(tz=timezone.utc)`

File: src/slurm_plugin/clustermgtd.py (hunk 1 above)
Comment (hanwen-pcluste) [reply src/slurm_plugin/clustermgtd.py:81]: Done

File: src/slurm_plugin/clustermgtd.py (hunk 2 above)
Comment (hanwen-pcluste) [reply src/slurm_plugin/clustermgtd.py:113]: Done

**Review by demartinofra — APPROVED** [review]

**Pull request merged**